shared/search/user-search/search-bar.desktop.js

```diff
@@ -100,8 +100,8 @@ class SearchBar extends Component<void, Props, void> {
             underlineShow={false}
             style={stylesInput}
             textStyle={{height: 31}} />
-          <Icon type='iconfont-remove' style={{marginRight: 16, opacity: this.props.searchText ? 1 : 0}}
-            onClick={() => this.refs.searchBox.clearValue()} />
+          {this.props.searchText && <Icon type='iconfont-remove' style={{marginRight: 16}}
```

Comment:
just hide this instead of being invisible., else we get bad wrapping at 800px of the hint text